### DISCUSSION TITLE
Advanced World Selection

### DISCUSSION THREAD

**Discussion by Omgise** [Roadhog360/Et-Futurum-Requiem#395] [Improvements]
I think porting the 1.8+ world selection interface would be great. :D

**Comment by Roadhog360** [comment]
What's different about 1.8+ world selection besides the icons?

**Comment by Omgise** [comment]
Graphical interface: Higher versions of the World Selection interface usually have a more modern and beautiful graphical design, and contain more detail and animation effects than lower versions.

Functional options: Higher versions will provide more functional options, such as custom world generation Settings, game mode selection, difficulty Settings, etc. This gives players more flexibility to adjust the game experience.

Types and types: The advanced version will introduce new world types or types, such as "Fine", "Flat", "Big Cave", etc., to provide more different play styles and challenges.

Map Preview: The advanced version will show a map preview, allowing you to see the general terrain and topography when selecting the world.

Filtering and sorting: The higher version will provide more filtering and sorting options, making it easier to find the world you want.

**Reply by Roadhog360** [reply]
These are either already in 1.7.10 or not in vanilla Minecraft at all. There is already world type selection and multiple other options.

What high quality graphics animations are you referring to? Vanilla never lets you review the seed or view its topography... Did ChatGPT write this?

**Reply by Omgise** [reply]
Yes😂

**Reply by Roadhog360** [reply]
Please suggest vanilla features only.
